src/Workspaces/Core/Portable/Diagnostics/DefaultAnalyzerAssemblyLoaderService.cs

```diff
@@ -1,40 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Composition;
-using System.IO;
-using Microsoft.CodeAnalysis.Host.Mef;
-using System.Collections.Immutable;
-using System.Collections.Generic;
-
-namespace Microsoft.CodeAnalysis.Host;
-
-[ExportWorkspaceServiceFactory(typeof(IAnalyzerAssemblyLoaderProvider)), Shared]
-[method: ImportingConstructor]
-[method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-internal sealed class DefaultAnalyzerAssemblyLoaderServiceFactory(
```

Comment:
this code moved to be next to IAnalyzerAssemblyLoaderProvider.  That's a standard pattern we have for workspace services.  We have hte interface, and the default impl next to it.